traffic_portal/app/src/common/modules/table/types/table.types.tpl.html

```diff
@@ -25,6 +25,19 @@
         <div class="pull-right">
             <button class="btn btn-primary" title="Create Type" ng-click="createType()"><i class="fa fa-plus"></i></button>
             <button class="btn btn-default" title="Refresh" ng-click="refresh()"><i class="fa fa-refresh"></i></button>
+            <div class="btn-group" role="group" title="Select Table Columns" uib-dropdown is-open="columnSettings.isopen">
+                <button type="button" class="btn btn-default dropdown-toggle" uib-dropdown-toggle aria-haspopup="true" aria-expanded="false">
+                    <i class="fa fa-columns"></i>&nbsp;
+                    <span class="caret"></span>
+                </button>
+                <ul ng-click="$event.stopPropagation()" class="column-settings dropdown-menu-right dropdown-menu" uib-dropdown-menu>
+                    <li role="menuitem" ng-repeat="c in columns">
```

Comment:
a `[role="menuitem"] must be the direct child of a `[role="menu"]` or `[role="menubar"]` - you can add these roles to the `<ul>`, or you can get that implicitly by using a [`<menu>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/menu) instead.